desktop/assets/texts/en-US/filesystem_access_dialog.ftl

```diff
@@ -0,0 +1,7 @@
+filesystem-access-dialog-title = Requesting Filesystem Access
+
+filesystem-access-dialog-message = The current movie is attempting to access the following file. Are you sure you want to allow it?
```

Comment:
nit: Change "Are you sure you want to allow it?" to "Do you want to allow it?"

The former assumes intent and feels like you're questioning the users intent, rather than asking the user what their intent is.


(perhaps change "access" to "read", too? These are read-only operations, right?)

Reply:
(If so, I suppose, the network/web permission dialogs should also be updated to be consistent.)

Reply:
Done, and yes, this is read-only access to the file (unless we implement AIR API)

Reply:
Network access / open url dialogs are updated here: https://github.com/ruffle-rs/ruffle/pull/18066